internal/ghrepo/repo.go

```diff
@@ -28,14 +28,15 @@ func FullName(r Interface) string {
 	return fmt.Sprintf("%s/%s", r.RepoOwner(), r.RepoName())
 }
 
-// FromFullName extracts the GitHub repository inforation from an "OWNER/REPO" string
-func FromFullName(nwo string) Interface {
+// FromFullName extracts the GitHub repository information from an "OWNER/REPO" string
+func FromFullName(nwo string) (Interface, error) {
 	var r ghRepo
 	parts := strings.SplitN(nwo, "/", 2)
-	if len(parts) == 2 {
-		r.owner, r.name = parts[0], parts[1]
+	if len(parts) != 2 {
```

Comment:
Could we also check that none of the parts are blank? i.e. `"/"`, `"a/"` and `"/b"` should all result in errors

Reply:
Good idea, additional validation added (although input welcome on the approach): f3f711f55cc5b7119a8372cf72cba47070e63812